lib/stats/tests/test_stats_prometheus.c

```diff
@@ -168,6 +168,13 @@ Test(stats_prometheus, test_prometheus_format_sanitize)
   stats_cluster_free(cluster);
 }
 
+void
+unix_time_set_now(UnixTime *unix_time)
+{
+  unix_time->ut_sec = 1e9;
+  unix_time->ut_usec = 1000;
+  unix_time->ut_gmtoff = 0;
+}
```

Comment:
there's a fake_time.{c,h} module in libtest that can fake the current timestamps using the syslog-ng cache,
instead of relying on linker tricks.

Reply:
Thanks, fixed.